Bottom: a60724bf5aa9081e8c88db5b5cfd3786af42b333
Top:    7877a1a5ec8b625a7127ef30ba236e4677df5df2
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 11:37:30 -0700

Refresh of fork

---

diff --git a/src/preload.c b/src/preload.c
index 731643d..a1073d5 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -846,7 +846,7 @@ pid_t fork(void)
 	sin6.sin6_flowinfo = sin6.sin6_scope_id = 0;
 	memset(&sin6.sin6_addr, 0, sizeof sin6.sin6_addr);
 
-	sem = sem_open("/rsocket_fork", O_CREAT, 0644, 1);
+	sem = sem_open("/rsocket_fork", O_CREAT | O_RDWR, S_IRWXU, 1);
 	if (sem == SEM_FAILED)
 		goto out;
